test/runtests.jl

```diff
@@ -5,6 +5,13 @@ using Base.Test
 include("abs_rev.jl")
 include("libieeep1788_rev.jl")
 
+include("Non1788tests/inv_hyperbolic.jl")
```

Comment:
I suggest adding a file in `Non1788tests` that just `include`s these tests, then just `include` that single file here.